halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/Main.java

```diff
@@ -24,8 +24,9 @@ public static void main(String[] args) {
 
     try {
       hal.execute();
-    } catch (RetrofitError e) {
+    } catch (Exception e) {
       AnsiUi.error(e.getMessage());
+      AnsiUi.remediation("Please rerun this command with -d, save the result, and post it in an issue on https://github.com/spinnaker/spinnaker/issues");
```

Comment:
I guess it could contain sensitive data - I'll drop the reference to `-d` and ask the user to post how to recreate their bug instead if possible.